components/menu/menu-item-styles.js

```diff
@@ -27,7 +25,23 @@ export const menuItemStyles = css`
 		z-index: 2;
 	}
 
-	:host([disabled]), :host([disabled]:hover), :host([disabled].focus-visible) {
+	/** separated because safari is having trouble parsing these */
```

Comment:
Maybe mention in these comments that it's just Safari <15.4 that has trouble with these entries? That way when/if we eventually stop caring about those versions we can decide to re-combine them.

Reply:
Yep, good point!